jax/_src/core.py

```diff
@@ -2003,7 +2003,7 @@ def _invalid_shape_error(shape: Shape, context: str=""):
     msg += f" {context}."
   if any(isinstance(x, Tracer) and isinstance(get_aval(x), ShapedArray)
          and not isinstance(get_aval(x), ConcreteArray) for x in shape):
-    msg += ("\nIf using `jit`, try using `static_argnums` or applying `jit` to "
+    msg += ("\nIf using jit, try using static_argnums or applying jit to "
```

Comment:
Changed this because the ipythontb lexer fails when it encounters backticks.

We've run into this before – I really should upstream a fix to `myst_nb.core.lexers:IPythonTracebackLexer`